src/components/Calendar/EventColorSelector/EventColorSelector.js

```diff
@@ -0,0 +1,187 @@
+import * as React from "react";
+import {
+    Avatar, 
+    InputLabel,
+    Select,
+    Stack,
+    TextField
+} from "@mui/material";
+import Typography from "@mui/material/Typography";
+import { PrimaryButton2 } from '../../CustomMUIComponents/CustomButtons';
+import MenuItem from "@mui/material/MenuItem";
+import CircleIcon from '@mui/icons-material/Circle';
+import axios from "axios";
+import { useNavigate } from "react-router";
+import { useState } from "react";
+import PersistentDrawerLeft from '../../NavDrawer/navDrawer'
+import { CustomWhiteCard } from '../../CustomMUIComponents/CustomCards';
+
+
+export default function EventColorSelector() {
+    const navigate = useNavigate();
+    const [errorMessage, setErrorMessage] = useState('')
+    const [eventSettingData, setEventSettingData] = React.useState({
+        title: '',
+        color: '',
+        avatarText: '',
+    });
+
+    React.useEffect(() => {
+        let user = localStorage.getItem("username")
+        setEventSettingData({ ...eventSettingData, owner: user })
+    })
+
+    React.useEffect(() => {
+        fetchData();
+    }, [])
+    function fetchData() {
+        axios.get('http://localhost:5000/friend/murth')
```

Comment:
use the processEnv url. you also need to parametrize the username you send in the get request.
axios.get(` ${process.env.REACT_APP_BASE_URL}friend/${username}`) (See calendar.js)